### DISCUSSION TITLE
opencd-python istallation filed

### DISCUSSION THREAD

**Discussion by FulvioFabiani** [g-api-org/discussions#8] [General]
On W10 with Python 3.11.5 istalled, i've done pip install opencv-python. i've istalled the 4.8.0.76 version, then i 've tried to import cd2 but this is the message:  
C:\Windows\system32>python
Python 3.11.5 (tags/v3.11.5:cce6ba9, Aug 24 2023, 14:38:34) [MSC v.1936 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import cv2
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Program Files\Python311\Lib\site-packages\cv2\__init__.py", line 181, in <module>
    bootstrap()
  File "C:\Program Files\Python311\Lib\site-packages\cv2\__init__.py", line 153, in bootstrap
    native_module = importlib.import_module("cv2")
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python311\Lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ImportError: DLL load failed while importing cv2: Impossibile trovare il modulo specificato.

How can i resolve the error?
Thanks

**Comment by dmatveev** [comment]
Hi @FulvioFabiani, thanks for report, I believe it is a common OpenCV problem so I filed a bug on that in the OpenCV Issue tracker: https://github.com/opencv/opencv/issues/24273

**Comment by dmatveev** [comment]
Closing the discussion as the bug was opened.
